src/wrapper/quatypesconverter.cpp

```diff
@@ -209,8 +209,8 @@ QMetaType::Type getQArrayType(const QMetaType::Type & type)
 	}
 	// TODO : check and use with QUaDataType::
 	auto strTypeName = QString(QMetaType::typeName(type));
-	strTypeName      = strTypeName.split("<").at(1);
-	strTypeName      = strTypeName.split(">").at(0);
+	strTypeName      = strTypeName.split(QChar(L'<')).at(1);
```

Comment:
Thanks for the PR, this is the way. 
Overall looks great.
Question, why here we need the `L`?

Reply:
With stricter project settings without "L" there will be an error. https://en.cppreference.com/w/cpp/language/string_literal

Reply:
It's probably better to replace QChar(L'<') with QLatin1Char('<').